dataprofiler/tests/profilers/utils.py

```diff
@@ -191,6 +191,8 @@ def assert_profiles_equal(actual, expected):
         # Condition to test whether the types are equal when a value can be float or float64
         if type(actual_value) is np.float64 or type(expected_value) is np.float64:
             assert type(float(actual_value)) == type(float(expected_value))
+        elif type(actual_value) is np.int64 or type(expected_value) is np.int64:
```

Comment:
this and the above line actually feels strange. why do we have to cast and why would it be different?//

Reply:
what is the condition that occurs when they aren't the same?/

Reply:
This was my initial gripe about the types. Initialization without any processed data is float and int types.
Then when we actually process (or load in) a profile they are converted to int64s and float64s. This happens on this test: `test_json_decode`

Reply:
But once we do the readin for data (like in `test_json_decode_after_update`) these conditions dont really do anything